everviz/plugins/lineplot/lineplot.py

```diff
@@ -0,0 +1,99 @@
+import os
+import pkg_resources
+
+import dash_html_components as html
+import dash_core_components as dcc
+
+import plotly.graph_objs as go
+import plotly.express as px
+from uuid import uuid4
+from dash.dependencies import Output, Input
+from webviz_config import WebvizPluginABC
+from webviz_config.webviz_assets import WEBVIZ_ASSETS
+from everviz.data.load_csv.get_data import get_data
+
+
+class Lineplot(WebvizPluginABC):
+    def __init__(self, app, data_path, title="Lineplot"):
+        super().__init__()
+        self.title = title
+        self.graph_id = f"graph-{uuid4()}"
+        self.filter_box_id = f"dropdown-{uuid4()}"
+        self.radio_id = f"dropdown-{uuid4()}"
+
+        self.data_path = data_path
+        self.set_callbacks(app)
+
+        ASSETS_DIR = pkg_resources.resource_filename("everviz", os.path.join("assets"))
+
+        WEBVIZ_ASSETS.add(os.path.join(ASSETS_DIR, "axis_customization.css"))
```

Comment:
When using the latest webviz-config I think only Path objects can be passed to `WEBVIZ_ASSETS`

Reply:
Thanks! Will update